_clients/index.md

```diff
@@ -26,6 +26,10 @@ For example, a 1.0.0 client works with an OpenSearch 1.1.0 cluster, but might no
 
 Most clients that work with Elasticsearch OSS 7.10.2 *should* work with OpenSearch, but the latest versions of those clients might include license or version checks that artificially break compatibility. This page includes recommendations around which versions of those clients to use for best compatibility with OpenSearch.
 
+{: .note}
+OpenSearch 2.0.0 no longer supports compatibility with legacy clients. Due to breaking changes with REST APIs, clients using older OpenSearch 1.x versions can't connect with OpenSearch 2.0.
```

Comment:
I think saying that OpenSearch 2.0 cannot connect to 1.x might not be correct. Except for the previously deprecated APIs that are removed in 2.0, all other APIs can be successfully used. @saratvemulapalli Please let me know if there is anything I missed here.

Reply:
Thanks @VachaShah , ok will follow up and ask for review by @saratvemulapalli  too.

Reply:
😄 Just a note, the inclusive naming changes (especially for changing "master" terminology) in OpenSearch 2.x will not impact the compatibility with OpenSearch 1.x clients.

Reply:
+1 to @VachaShah 
OpenSearch 1.x clients should be able to communicate with OpenSearch 2.0 except for the breaking changes we made: https://github.com/opensearch-project/OpenSearch/issues/2480

I would probably change it as, _ Due to breaking changes with REST APIs, some features are not supporting when using OpenSearch 1.x clients to connect with OpenSearch 2.0_

Reply:
great, thanks everyone for the details.